documentation/projects/proposals/document_all_media_properties/20230307-project_proposal.md

```diff
@@ -1,4 +1,5 @@
-Project proposal for
+# 2023-03-21 Project proposal for
+
 [Document all media properties](https://github.com/WordPress/openverse/issues/412)
```

Comment:
This makes the title fall on a second line: 
![image](https://user-images.githubusercontent.com/10214785/235811679-c288b24b-168b-488c-8c10-3e46e235a30f.png)

Perhaps we could add this link at the top of the description?